content/03.results-and-discussion.md

```diff
@@ -15,33 +15,33 @@ nf-core/eager provides improved efficiency over the original EAGER pipeline by r
+In nf-core/eager, all tools also originally used in EAGER have been updated to latest versions, as available on Bioconda (@doi:10.1038/s41592-018-0046-7) and conda-forge (@url:https://conda-forge.github.io) to ensure widespread accessibility and stability of utilized tools. The MapDamage2 (@doi:10.1093/bioinformatics/btt193) and Schmutzi (@doi:10.1186/s13059-015-0776-0) methods have not been carried over to nf-core/eager, the first because a more performant successor method (DamageProfiler) exists, and the latter because a stable release of the method could not be migrated to Bioconda. We anticipate that there will be an updated version of Schmutzi in the near future that will allow us to integrate the method again in nf-core/eager, once a version is released on Bioconda. Support for Bowtie2 (@doi:10.1038/nmeth.1923) will be added in the near future, after consultation with the palaeogenetics community. New tools to the basic workflow include fastp (@doi:10.1093/bioinformatics/bty560) for the removal of poly-G sequencing artefacts that are common in 2-colour Illumina sequencing machines (such as the increasingly popular NextSeq and NovaSeq platforms). We have also included the FreeBayes genotyper as an alternative to the human-focused GATK tools. We have also maintained the possibility of using the now unsupported GATK UnifiedGenotyper, as the GATK HaplotypeCaller performs _de novo_ assembly around possible variants, which may not be suitable for low-coverage aDNA data.
 
-We have further extended functionality of the pipeline, primarily focusing on ancient metagenomic analysis to be run alongside the standard genomic analysis against a single reference genome. We have added the ability to screen all off-target reads (from mapping to the supplied reference FASTA) with two metagenomic profilers: MALT (@doi:10.1101/050559, @doi:10.1038/s41559-017-0446-6) and Kraken2 (@doi:10.1186/s13059-019-1891-0). Characterisation of properties of authentic ancient DNA (@doi:10.1073/pnas.0704665104) from MALT alignments is carried out with the HOPS pipeline (@doi:10.1186/s13059-019-1903-0). Ancient metagenomic studies sometimes may include comparative samples from living day individuals (@doi:10.1186/s40168-019-0717-3). To support open data, whilst respecting data privacy, nf-core/eager includes a 'strip_fastq' script (by M.B.) which creates raw FASTQ files but with reference-genome mapped reads removed. This then allows safe upload of sequencing data to public repositories with identifiable data removed.
```

Comment:
Why do you want to remove the initials?

Reply:
Is that the normal way to do this in bioinfo papers? seems strange to me that the specific scripts are attributed to collaborators in the main text. That is what the "Author contributions" section is for in papers.

Reply:
Yeah normally thats not used - either make people co-author (which all of them are) or have them for very minor stuff in acknowledgments ;-)

Reply:
Ok fair enough. Was trying to give more direct attribution to 'unpublished' tools, but I guess not necessary.